docs/podman-diff.1.md

```diff
@@ -20,13 +20,16 @@ Alter the output into a different format.  The only valid format for diff is `js
 
 ## EXAMPLE
 
-podman diff redis:alpine
+```
+# podman diff redis:alpine
```

Comment:
most of our examples don't have the default root prompt in it, did you mean to add this here and at line 32?  Does this require root?  I'd drop it unless it does.

Reply:
How does this tie into being root? I just added the # in front to differentiate between the command and the results. The other docs do it as well - examples are inspect, login.

Reply:
```
# podman ...
```
Indicates podman running as root.  `#` is the default prompt given by bash when you become root.

Reply:
oh,  I never realised that.  A lot of our docs do that and you need to be root to run podman diff.

Reply:
What @rhatdan said.  We've run into this from time to time.  There are people that upon seeing the '#' at the front of an example assume that the command requires root prvis.  It's best to remove them unless root is required and yep, a number of other podman pages have them in play that probably should not.

Reply:
@TomSweeneyRedHat  got it! The diff and inspect commands require root privileges to run. I can look at the other docs and fix them if they are incorrectly using the #.